### ISSUE TITLE
Feature Idea: "batch import" text files

### ISSUE THREAD

**Issue by beren6** [olivierkes/manuskript#351]
Hi there!

First of all, thanks so much for your work on this tool--it's a truly brilliant piece of software, and the plain-text storage format plus other unique features make it beat Scr*** hands-down! 

I had an idea for a feature I thought I'd share (I'm not a programmer, so i don't know how feasible this would be; it'd just be a feature that I'd personally find very useful): right now Mansukript has the ability to import individual text files into its framework. I'm wondering if it would be feasible to implement a "batch import" feature, that would allow one to import, say, a folder-full of individual text files all at once, each of which would become its own "card" on the corkboard. This'd be really helpful, for example, for projects where one has been working with many individual text files that one would like to continue working on in Manuskript.

Just a thought!

Mark

**Comment by plum** [comment]
Yes, I can see that this is a really well thought out, much-needed software package. It is just the ticket (and I've tried many over the years).
I see in my installation (latest versionfrom git on archlinux via the aur repository), that there is in fact an opeuin to import a directory, obtainable after selecting import. 
I triess this - and I have many .txt files of journal notes to now layout and work up. Unfortunately manuskript crashes each time, shortly after selcting the required directory.

I should offer debug info - not sure where to obtain this at the pesent time - give me a hint and I'll post it.
Many thanks,
JOhn

**Comment by gedakc** [comment]
@plum there have been several enhancements made to Manuskript since the old [ArchLinux AUR Package](https://aur.archlinux.org/packages/manuskript-git/) 0.7.0.r0.g97e4679-1 version.

Before reporting a bug please try at least the 0.8.0 official manuskript release, or even better the **develop** branch and [Run Manuskript from Source Code on Linux](https://github.com/olivierkes/manuskript/wiki/Run-Manuskript-from-Source-Code-on-Linux).

**Comment by plum** [comment]
Thanks for prompt reply Gedakc. I installed and ran from git source, and found that it was the devel branch. Same crash happens,  after first ensuring all files in floder were .md extensions woith plain text contents.
How do I get a useful crash report to you?

**Comment by gedakc** [comment]
To capture the console log, start _manuskript_ from a terminal window and perform the steps that cause the crash.  Then you can copy the console log and paste it into this report.  To keep the formatting, place triple single quotes at the beginning and end of the console log.  Please also describe the steps you use to reproduce the crash.

**Comment by plum** [comment]
tahnks Gedakc, I will do that.
